### PR TITLE
adds contrast property to textArea component

### PR DESCRIPTION
**Pull request by ValbertMartins**
Describe your changes here.

• add contrast property to textarea component (same property existent in textInput component)

### Screenshots
• textarea with contrast 
Captura de tela 2023-07-04 195205

### Merge checklist

- [ ] Added/updated tests
- [x] Added/updated documentation
- [ ] Changes are [SSR compatible](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#ssr-compatibility)
- [X] Tested in Chrome
- [ ] Tested in Firefox
- [ ] Tested in Safari
- [ ] Tested in Edge

Take a look at the [What we look for in reviews](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#what-we-look-for-in-reviews) section of the contributing guidelines for more information on how we review PRs.

### CODE REVIEW

**Comment by broccolinisoup** [comment]
Hi @ValbertMartins 👋🏻 Thanks for the PR! We usually need an issue for these type of PRs that introduces feature requests to be able to access it from an accessibility, usability etc point of view. Would you mind creating an [issue](https://github.com/primer/react/issues/new/choose) so that we can discuss it as a team?

**Review by broccolinisoup — COMMENTED** [review]
Hi @ValbertMartins 👋🏻 Thanks for the PR! We usually need an [issue](https://github.com/primer/react/issues/new/choose) for these type of PRs that introduces feature requests to be able to access it from an accessibility, usability etc point of view. Would you mind creating an issue so that we can discuss it as a team?

**Comment by lesliecdubs** [comment]
Sorry this got closed as stale, we just re-triaged the issue https://github.com/primer/react/issues/3487 related to this PR and think this is still valid. Reopening and requesting review from maintainers 🔄

File: src/Textarea/Textarea.tsx [hunk 1]
```diff
@@ -53,7 +57,7 @@ const StyledTextarea = styled.textarea<TextareaProps>`
     css`
       resize: none;
     `}
-
+
```
Comment (broccolinisoup) [inline src/Textarea/Textarea.tsx:60]:
```suggestion
```

**Review by broccolinisoup — COMMENTED** [review]
Hello @ValbertMartins 👋🏻 Thanks so much for pushing this PR! Could you please add a changeset for the changes we are introducing here? You can find the instructions in [our contributing documentation](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#adding-changeset-to-your-pull-request) - Please let me know if you have any questions 🙌🏻

**Comment by lesliecdubs** [comment]
@broccolinisoup would you be willing to take over this PR to [add in a changeset](https://github.com/primer/react/pull/3484#pullrequestreview-1623423680) so we can get this merged, unless there's more outstanding to do?

@ValbertMartins you are welcome to make this change ^ as well, just didn't want to let this PR stagnate again!

**Review by broccolinisoup — APPROVED** [review]
Looks great! 🚀 Just added the changeset file, thanks so much for raising this PR.

**Comment by broccolinisoup** [comment]
Thanks @lesliecdubs for bringing this PR to my attention! I added the changeset file, and reviewed the PR. All looks great.

@ValbertMartins I'm going to merge it now, thanks for your contribution and sorry for the delay! 🙏🏻

**Pull request merged**